lib/kafo/kafo_configure.rb

```diff
@@ -375,6 +385,7 @@ def unset
 
     def config_file
       return CONFIG_FILE if defined?(CONFIG_FILE) && File.exists?(CONFIG_FILE)
+      return self.class.scenario_manager.select_scenario if self.class.scenario_manager.configured?
```

Comment:
I changed the way scenario manager prints the errors. It should be more user friendly now. It prints the message using `say` and logs the same in `error` debug level for cases when logging is set up already. 

I found implementation of printing `--help` with no config file non-trivial. If it is acceptable I would fix it in PR that will follow and will include refactoring bringing in better isolation of `KafoConfigure` and `Configuration` classes. My plan is to pass along Configuration objects rather then the config_file filenames, which would allow us to create e.g.  "dummy" configuration with no modules to finish `KafoConfigure` initialization and printing help. It should make testing easier as well.